.cirrus.star

```diff
@@ -0,0 +1,9 @@
+load("github.com/SonarSource/cirrus-modules@v2", "load_features")
+load("cirrus", "env", "fs", "yaml")
+
+
+def main(ctx):
+    if env.get("CIRRUS_BRANCH") == env.get("CIRRUS_DEFAULT_BRANCH"):
+        return yaml.dumps(load_features(ctx, only_if=dict())) + fs.read(".cirrus-private.yml")
+    else:
+        return fs.read(".cirrus.yml")
```

Comment:
You should probably just return contents of `.cirrus-private.yml` which will get appended to `.cirrus.yml`

```suggestion
    if env.get("CIRRUS_BRANCH") == env.get("CIRRUS_DEFAULT_BRANCH"):
        return fs.read(".cirrus-private.yml")
```

Reply:
Oh, thank you @fkorotkov! I need actually to rename `.cirrus.yml` in order to get its content only for PRs, not for master.